README.md

```diff
@@ -1 +1,68 @@
-# gleaming-dragonfly-calculator
\ No newline at end of file
+#  Mac Calculator Clone
+## Team:
+
+Frankie Rocha
+
+Project: [#150-Mac Calculator Clone](http://jsdev.learnersguild.org/goals/150-Mac_Calculator_Clone.html)
+
+Team Name: gleaming-dragonfly
+
+## Stage 1
+
+**Specs**
+
+ - [x] All text is in the `Roboto` web font
+ - [x]  Your HTML and CSS follows this style guide
+ - [x]  The calculator is positioned in the center of the page, both vertically and horizontally
+ - [x]  The calculator is a fixed size. It does not change in size when the page resizes.
+ - [ ]  If the window is too small for the calculator, the page scrolls, both vertically and horizontally
+ - [x]  Each button has a CSS transition to slightly darken the background color on hover over 100ms
+ - [x]  Each button has a CSS transition to slightly darken the background color on click over 100ms
+ - [x]  All class names re: the calculator are name-spaced under .calculator-…
```

Comment:
the short answer is: all classnames are like global variables. You only get one and they're applied everywhere. CSS does not have any scoping features. So if you use a classname like `size-2` it is a global classname that can be applied anywhere despite the fact that you intended it to only be used within a `.calculator`. The prefix communicated to other developers that this is specific to a `.calculator`. 

Think about a huge website like linkedIn. If everyone created any old classname they wanted they would step on each other. So we need something like prefixes or even something more complex like BEM to prevent a huge mess of unorganized CSS